swan-lake/resources/featured-scenarios/build-a-data-service-in-ballerina.md

```diff
@@ -59,224 +55,229 @@ CREATE TABLE Company.Employees (
+
+```json
+{    
+    "first_name": "test",
+    "last_name": "test",
+    "email": "test@test.com",
+    "phone": "882 771 110",
+    "job_title": "Sales Manager"
+}
+```
+
+![Create data record](/learn/images/featured-scenarios/build-a-data-service-in-ballerina/create-data-record.gif)
+
+The completed record will be as follows.
+
 ```ballerina
 import ballerina/time;
 
-public type Employee record {|
-    int employee_id?;
+type Employee record {|
     string first_name;
     string last_name;
     string email;
     string phone;
-    time:Date hire_date;
-    int? manager_id;
     string job_title;
+    int employee_id?;
+    int? manager_id;
+    time:Date hire_date;
 |};
 ```
 
-### Add the MySQL driver
-
-The MySQL driver JAR is necessary to connect to and interact with a MySQL server. Select one out of the methods below to add it.
-
-1. Import the `ballerinax/mysql.driver` package in your `main.bal` file. This package bundles the latest MySQL driver so that the MySQL connector can be used in Ballerina packages easily.
-
-   ```ballerina
-   import ballerinax/mysql.driver as _;
-   ```
-   
-2. Update the `Ballerina.toml` file with the Maven dependency params for the MySQL driver.
-   ```toml
-   [[platform.java11.dependency]]
-   groupId = "mysql"
-   artifactId = "mysql-connector-java"
-   version = "8.0.26"
-   ```
-   
-3. Download the driver JAR manually and update the path in the `Ballerina.toml` file
-   ```toml
-   [[platform.java11.dependency]]
-   path = "/path/to/mysql/driver.jar”
-   ```
-   
 ### Define MySQL configurations
 
-In the package directory, create a new file named `Config.toml` and specify the configurations below to connect to the MySQL database.
+Follow the steps below to define the MySQL configurations.
 
-```toml
-USER="root"
-PASSWORD="rootPassword"
-HOST="localhost"
-PORT=3306
-DATABASE="Company"
-```
+1. In the package directory, create a new file named `Config.toml` and specify the configurations below to connect to the MySQL database.
 
-To redefine the above variables and access them from within your Ballerina program, add the code below in the `main.bal` file.
+    ```toml
+    USER="root"
+    PASSWORD="rootPassword"
+    HOST="localhost"
+    PORT=3306
+    DATABASE="Company"
+    ```
 
-```ballerina
-configurable string USER = ?;
-configurable string PASSWORD = ?;
-configurable string HOST = ?;
-configurable int PORT = ?;
-configurable string DATABASE = ?;
-```
+2. Define configurable variables to redefine the above variables and access them from within your Ballerina program.
+
+    >**Info:** For more information on defining configurable variables in Ballerina, see [Provide values to configurable variables](/learn/provide-values-to-configurable-variables/).
+
```

Comment:
Let's not add default values to the configurable variables. We can keep them as ?

Reply:
Updated as follows.

```
configurable string user = ?;
configurable string password = ?;
configurable string host = ?;
configurable int port = ?;
configurable string database = ?;
```